README.md

```diff
@@ -31,6 +31,7 @@ Suppose the forked repository is `https://github.com/your-repo/parser`.
 
     ```
     GO111MODULE=on go mod edit -replace github.com/pingcap/parser=github.com/your-repo/parser@your-branch
+    GO111MODULE=on go mod tidy
```

Comment:
The parser repo doesn't check `tidy`, `tidy` check is used in TiDB
So this line should be moved to line 59

Reply:
https://github.com/pingcap/parser/pull/64/files#diff-04c6e90faac2675aa89e2176d2eec7d8L57